src/SFA.DAS.EmployerAccounts.Web/ViewModels/CohortV2ViewModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using SFA.DAS.EmployerAccounts.Models.Commitments;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.EmployerAccounts.Web.ViewModels
+{   
+ 
+
+    public class CohortV2ViewModel
+    {  
+        public long CohortId { get; set; }        
+        public int? NumberOfDraftApprentices { get; set; }
+        public string HashedCohortReference { get; set; }
+        public string HashedDraftApprenticeshipId { get; set; }
+        public CohortStatus CohortStatus { get; set; }
+        public virtual ICollection<ApprenticeshipViewModel> Apprenticeships { get; set; } = new List<ApprenticeshipViewModel>();
+        public bool HasSingleDraftApprenticeship =>  NumberOfDraftApprentices == 1 &&  Apprenticeships.Single().ApprenticeshipStatus == ApprenticeshipStatus.Draft ; //CohortsCount == 1 && NumberOfDraftApprentices == 1 && ApprenticeshipsCount == 0;        
+        public bool HasApprovedApprenticeship => Apprenticeships.First().ApprenticeshipStatus == ApprenticeshipStatus.Approved;
+        public string ViewOrEditApprenticeDetails => $"unapproved/{HashedCohortReference}/apprentices/{HashedDraftApprenticeshipId}";
+        public string ApprovedOrRejectApprenticeDetails => $"unapproved/{HashedCohortReference}";
```

Comment:
This should be on the ApprenticeshipViewModel since it is related to the Apprenticeship.

Reply:
done